easybuild/easyconfigs/p/PyTables/PyTables-3.3.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   Copyright 2014-2017 adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# License::     BSD
+#
+# Notes::
+##
+
+easyblock = 'PythonPackage'
+
+name = 'PyTables'
+version = '3.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.pytables.org'
+description = """PyTables is a package for managing hierarchical datasets and designed to efficiently and easily cope
+ with extremely large amounts of data. PyTables is built on top of the HDF5 library, using the Python language and the
+ NumPy package. It features an object-oriented interface that, combined with C extensions for the performance-critical
+ parts of the code (generated using Cython), makes it a fast, yet extremely easy to use tool for interactively browse,
+ process and search very large amounts of data. One important feature of PyTables is that it optimizes memory and disk
+ resources so that data takes much less space (specially if on-flight compression is used) than other solutions such as
+ relational or object oriented databases."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/PyTables/PyTables/archive/']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('numexpr', '2.6.1', versionsuffix),
+    ('HDF5', '1.8.18'),
```

Comment:
@verdurin hmm, just wonder why travis didn't pick this one up, may i ask where did you pull out the test report? Thx